classes/PodsTermSplitting.php

```diff
@@ -0,0 +1,515 @@
+				"
+				UPDATE
+				    {$wpdb->commentmeta}
+				SET
+					meta_value = REPLACE( meta_value, %s, %s )
+				WHERE
+				    meta_key = %s
+					AND meta_value LIKE '%%%s%%'
+				",
+				$target_serialized,
+				$replace_serialized,
+				$meta_key,
+				pods_sanitize_like( $target_serialized )
+			) );
+
+			$this->append_progress( $task );
+		}
+
+	}
+
+	/**
+	 * Called for all fields related to the target taxonomy that are in a user Pod
+	 *
+	 * @param string $field_name
+	 */
+	private function update_usermeta( $field_name ) {
+
+		/** @global wpdb $wpdb */
+		global $wpdb;
+
+		// Fix up the unserialized data
+		$task = "update_usermeta_{$field_name}_unserialized";
+		if ( ! $this->have_done( $task ) ) {
+
+			$table = $wpdb->usermeta;
+			$data = array( 'meta_value' => $this->new_term_id );
+			$where = array(
+				'meta_key'   => $field_name,
+				'meta_value' => $this->term_id
+			);
+			$format = '%s';
+			$where_format = array( '%s', '%s' );
+			$wpdb->update( $table, $data, $where, $format, $where_format );
+
+			$this->append_progress( $task );
+		}
+
+		// Fix up the serialized data
+		$task = "update_usermeta_{$field_name}_serialized";
+		if ( ! $this->have_done( $task ) ) {
+
+			$meta_key = sprintf( '_pods_%s', $field_name );
+			$target_serialized = sprintf( ';i:%s;', $this->term_id );
+			$replace_serialized = sprintf( ';i:%s;', $this->new_term_id );
+
+			$wpdb->query( $wpdb->prepare(
+				"
+				UPDATE
+				    {$wpdb->usermeta}
+				SET
+					meta_value = REPLACE( meta_value, %s, %s )
+				WHERE
+				    meta_key = %s
+					AND meta_value LIKE '%%%s%%'
+				",
+				$target_serialized,
+				$replace_serialized,
+				$meta_key,
+				pods_sanitize_like( $target_serialized )
+			) );
+
+			$this->append_progress( $task );
+		}
+
+	}
+
+	/**
+	 * Called for all fields related to the target taxonomy that are in a user Pod
+	 *
+	 * @param string $pod_name
+	 * @param string $field_name
+	 */
+	private function update_setting_meta( $pod_name, $field_name ) {
+
+		/** @global wpdb $wpdb */
+		global $wpdb;
+
+		$option_name = "{$pod_name}_{$field_name}";
+
+		// Fix up the unserialized data
+		$task = "update_setting_meta_{$pod_name}_{$field_name}_unserialized";
+		if ( ! $this->have_done( $task ) ) {
+
+			// UPDATE {$wpdb->options} SET option_value = '{$new_term_id}' WHERE option_name = '{$pod_name}_{$field_name}' AND option_value = '{$term_id}'
+			$table = $wpdb->options;
+			$data = array( 'option_value' => $this->new_term_id );
+			$where = array(
+				'option_name'  => $option_name,
+				'option_value' => $this->term_id
+			);
+			$format = '%s';
+			$where_format = array( '%s', '%s' );
+			$wpdb->update( $table, $data, $where, $format, $where_format );
+
+			$this->append_progress( $task );
+		}
+
+		// Fix up the serialized data
+		$task = "update_setting_meta_{$pod_name}_{$field_name}_serialized";
+		if ( ! $this->have_done( $task ) ) {
+
+			$target_serialized = sprintf( ';i:%s;', $this->term_id );
```

Comment:
Integer is how it's to be interpreted when unserialized.  It is still stored as a string in the database and thus we need to build a string.